src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -93,14 +93,19 @@
  * uncaught exception handler}.
  *
  * @implNote
- * Conventionally, the native code that has launched the JVM will invoke the JNI function
- * <a href="{@docRoot}/../specs/jni/invocation.html#destroyjavavm">DestroyJavaVM</a>.
- * This function is responsible for initiating the shutdown sequence when the number of running
- * ({@linkplain Thread#isAlive() live}) non-daemon threads first drops to zero. When the shutdown
- * sequence completes and the JVM terminates, control is returned to native invocation code.
+ * Native code typically uses the
+ * <a href="{@docRoot}/../specs/jni/invocation.html">JNI Invocation API</a>
+ * to control launching and termination of the JVM. Such native code invokes the
+ * <a href="{@docRoot}/../specs/jni/invocation.html#jni_createjavavm">{@code JNI_CreateJavaVM}</a>
+ * function to launch the JVM. Subsequently, the native code invokes the
+ * <a href="{@docRoot}/../specs/jni/invocation.html#destroyjavavm">{@code DestroyJavaVM}</a>
+ * function to await termination of that JVM. The {@code DestroyJavaVM} function is responsible for
+ * initiating the shutdown sequence when the number of {@linkplain Thread#isAlive() running}
+ * non-daemon threads first drops to zero. When the shutdown sequence completes and the JVM
+ * terminates, control is returned to the native code that invoked {@code DestroyJavaVM}.
  * <p>
- * In typical JVM implementations, calling {@link #exit exit} or {@link #halt halt} will also
- * terminate the OS process hosting the JVM.
+ * This differs from the {@link #exit exit} or {@link #halt halt} methods. These methods typically
```

Comment:
In a new paragraph it is not clear what "This" refers to. I would run this into the previous paragraph as:

> This behavior differs from the {@link #exit exit} or {@link #halt halt} methods, which typically terminate the OS process hosting the JVM and do not interact with the JNI Invocation API.